lib/builders/observability-builder.ts

```diff
@@ -1,24 +1,26 @@
-import * as blueprints from '../../lib';
+import * as blueprintbuilder from '../../lib/stacks';
+import * as addons from '../../lib/addons';
 import * as utils from "../utils";
+import * as spi from '../../lib/spi'
```

Comment:
`make lint` - I see GH actions missing semicolons and such

Reply:
fixed